templates/windows/chocolateyInstall.ps1.erb

```diff
@@ -1,26 +1,40 @@
 $toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+$fileList = Join-Path -path $toolsDir -childpath "file-list.txt"
 
-<%- dirnames = get_directories.map {|d| d.path } -%>
-<%- dirnames.each do |dir| -%>
-  <%- windows_path = platform.convert_to_windows_path(dir) -%>
-if (Test-Path -path "<%= windows_path %>") {
-  Write-Debug "Warning: copying over '<%= windows_path %>'"
-} else {
-  New-Item -ItemType directory -Path "<%= windows_path %>"
+if (!(Test-Path -path "$fileList")) {
+  Write-Host "Unable to find file '$fileList', cannot proceed with install"
+  exit
 }
-$origin = Join-Path -path "$toolsDir" -childpath "<%= dir.gsub('/', '\\') %>"
-Copy-Item -path "$origin\*" -destination "<%= windows_path.gsub('/', '\\') %>\" -Recurse -Force
-<%- end -%>
 
-<%- filenames = get_files.map {|f| f.path} -%>
-<%- configfilenames = get_configfiles.map {|cf| cf.path} -%>
-<%- (filenames + configfilenames).each do |file| -%>
-  <%- windows_path = platform.convert_to_windows_path(file) -%>
-if (Test-Path -path "<%= windows_path %>") {
-  Write-Debug "Warning: copying over '<%= windows_path %>'"
-} elseif (!(Test-Path -path "<%= File.dirname windows_path %>")) {
-  New-Item -ItemType directory -Path "<%= File.dirname windows_path %>"
+$reader = [System.IO.File]::OpenText("$fileList")
+try {
+  for (;;) {
+    $destination = $reader.ReadLine()
+    if ($destination -eq $null) { break }
+    $originFile = "$destination" -replace '^[a-zA-Z]:/',''
+    $origin = Join-Path -path "$toolsDir" -childpath "$originFile"
+    if (Test-Path -path "$origin") {
```

Comment:
That seems like an ok approach, a file that's missing but in the file list seems like a pretty unlikely bug in packaging. I'd like to see a comment about that assumption in the script.